src/main/resources/db/migration/V1__user_create.sql

```diff
@@ -0,0 +1,73 @@
+-- USERS
+DROP TABLE IF EXISTS `user_bsierra1`;
+CREATE TABLE `user_bsierra1` (
+    `id` bigint(20) NOT NULL AUTO_INCREMENT,
+    `first_name` varchar(50) NOT NULL,
+    `email` varchar(100) NOT NULL UNIQUE,
+    `password_hash` varchar(64) NOT NULL,
+    `created_at` timestamp,
+    PRIMARY KEY (`id`)
+);
+
+create UNIQUE INDEX `user_email_unique` on user_bsierra1(email);
+
+-- SECTION
+DROP TABLE IF EXISTS `section_bsierra1`;
+CREATE TABLE `section_bsierra1` (
+                                    `id` bigint(20) NOT NULL AUTO_INCREMENT,
```

Comment:
These don't look formatted right (too much spacing). It would be helpful if you ran "Reformat File` in intellij